docs/reference/analysis.asciidoc

```diff
@@ -4,141 +4,42 @@
 [partintro]
 --
 
-_Text analysis_ is the process of converting text, like the body of any email,
-into _tokens_ or _terms_ which are added to the inverted index for searching.
-Analysis is performed by an <<analysis-analyzers,_analyzer_>> which can be
-either a built-in analyzer or a <<analysis-custom-analyzer,`custom`>> analyzer
-defined per index.
+_Text analysis_ is the process of converting unstructured text, like
+the body of an email or a product description, into a structured format that's
+optimized for search.
 
 [float]
-== Index time analysis
+[[when-to-configure-analysis]]
+=== When to configure text analysis
 
-For instance, at index time the built-in <<english-analyzer,`english`>> _analyzer_ 
-will first convert the sentence:
+{es} comes with smart defaults for text analysis. These defaults work well for
+most use cases and require no configuration.
 
-[source,text]
-------
-"The QUICK brown foxes jumped over the lazy dog!"
-------
+If you're happy with your search experience, no further setup is needed;
+you can skip the chapters in this section.
 
-into distinct tokens. It will then lowercase each token, remove frequent
-stopwords ("the") and reduce the terms to their word stems (foxes -> fox,
-jumped -> jump, lazy -> lazi). In the end, the following terms will be added
-to the inverted index:
+However, there are less common cases where configuring text analysis is
```

Comment:
I don't think that configuring an analyzer is an uncommon use case,  I would think it should be quite common for text fields.

Reply:
Good point! It's clearer if we just directly state that that if you use `text` fields, take a look. If not, go ahead and skip this section.

Made those changes with a2f08d21e0a93a245e19254118e4ece83d06d080.